app/models/course.rb

```diff
@@ -417,6 +426,13 @@ def reorder_weeks
     end
   end
 
+  # Makes sure that the home wiki
+  # is always a part of courses wikis.
+  def ensure_home_wiki_in_courses_wikis
+    return if home_wiki.nil?
```

Comment:
home_wiki shouldn't ever be nil, I don't think. Is this necessary?

Reply:
It is nil in the before_create callback because, it is before creation and there isn't a home_wiki yet.